src/cmds/exit.c

```diff
@@ -24,4 +24,6 @@ int main(int argc, char *argv[]) {
 	}
 
 	exit(ecode);
+
```

Comment:
Shouldn't `exit` be marked as `noreturn` instead?

Reply:
Remove this commit